R/modify-tree.R

```diff
@@ -8,7 +8,8 @@
 #' @param leaf A function applied to each leaf.
 #' @param is_leaf A predicate function that determines whether an element is
 #'   a leaf (by returning `FALSE`) or a node (by returning `FALSE`). The
-#'   default value, `NULL`, treats lists as nodes and everything else as leaves.
+#'   default value, `NULL`, treats bare lists as nodes and everything else
```

Comment:
Link to `vec_is_list()`?

Reply:
I'm going to leave this as is since once I switch to `is_node` instead of `is_leaf` the default will become `vec_is_list()` and then it's obvious.